pandas/tests/series/test_operators.py

```diff
@@ -91,6 +91,9 @@ def check(series, other, check_reverse=False):
         check(self.ts, 5, check_reverse=True)
         check(tm.makeFloatSeries(), tm.makeFloatSeries(), check_reverse=True)
 
+    def test_pos(self):
+        assert_series_equal(+self.series, self.series)
```

Comment:
Sorry, I'm new to contributing to this Pandas. Are there any other datatypes that I need to write test cases for other than series and dataframes? 

I'll write these tests and resubmit tomorrow.